content-services/content-sources-backend
# Fixes 3600: delete distributions and env on template delete

## Summary
Adds additional actions to the DeleteTemplate task so that the associated pulp distributions and candlepin environment are also deleted.

## Testing steps
1. Create a template with a mix of red hat and custom repositories
2. You can verify the existence of the distributions by browsing to `http://pulp.content:8080/pulp/content/<domain>/templates/<template_uuid>/`
3. You can verify the existence of the environment by GET requesting `http://localhost:8181/candlepin/environments/<env id>`. The env_id is the template_uuid without the hyphens.
4. Delete the template
5. Recheck those endpoints and you should get a 404.

## Checklist
